src/util/aws-util.ts

```diff
@@ -97,6 +139,17 @@ export class AwsUtil {
         return AwsUtil.masterAccountId;
     }
 
+    public static async GetGovCloudMasterAccountId(): Promise<string> {
+        if (AwsUtil.masterGovCloudAccountId !== undefined) {
+            return AwsUtil.masterGovCloudAccountId;
+        }
+        const govCloudCredentials = await AwsUtil.GetGovCloudCredentials();
+        const stsClient = new STS({ credentials: govCloudCredentials, region: 'us-gov-west-1' }); // if not set, assume build process runs in master
```

Comment:
its never really set elsewhere right? lets look at this on a second pass.
I think `--partition-master-account 123123123123` and `--partition us-gov` could/should indicate that `123123123123` is the management account for `us-gov`

this seems to be quite complicated to build and something we could easily add later.